pkg/pillar/hypervisor/kvm.go

```diff
@@ -441,8 +443,29 @@ func (ctx kvmContext) CreateDomConfig(domainName string, config types.DomainConf
 	diskContext := struct {
 		Machine               string
 		PCIId, DiskID, SATAId int
+		AioType               string
 		types.DiskStatus
-	}{Machine: ctx.devicemodel, PCIId: 4, DiskID: 0, SATAId: 0}
+	}{Machine: ctx.devicemodel, PCIId: 4, DiskID: 0, SATAId: 0, AioType: "threads"}
+
+	var osver []string
+	var major, minor, patch int
+
+	osver = strings.SplitN(getOsVersion(), ".", 3)
+
+	if len(osver) >= 1 {
+		major, _ = strconv.Atoi(osver[0])
+	}
+	if len(osver) >= 2 {
+		minor, _ = strconv.Atoi(osver[1])
+	}
+	if len(osver) >= 3 {
+		patch, _ = strconv.Atoi(osver[2])
+	}
+
+	if major >= 5 && minor >= 4 && patch >= 72 {
```

Comment:
Two suggestions here @temerkhanov 
   1. could you please turn 5.4.72 into something like a `const minUringKernelVersion = [5, 4, 72]` at the top of this file (feel free to come up with better name)?
   2. let's make the comparison real (jut to make sure that should we upgrade to 5.10.0 this check will still pass)?